src/main/java/github/mrh0/beekeeping/datagen/ItemModelGenerator.java

```diff
@@ -40,19 +36,20 @@ protected void registerModels() {
     }
 
     private ItemModelBuilder simpleItem(Item item) {
-        return withExistingParent(item.getRegistryName().getPath(),
+        return withExistingParent(item.toString(),
```

Comment:
Not sure this is correct

Reply:
Same with the stuff below

Reply:
I believe they would work as the implementation of the `item.toString` is:
```
   public String toString() {
      return Registry.ITEM.getKey(this).getPath();
   }
```

Though this is possibly a little fragile and prone to breaking in future changes, i've replaced it with forge registry calls now.